README.md

```diff
@@ -562,6 +562,51 @@ Use it to type in your key and install.
 Go back to Step 4A-4 and then finish with Step 4A-5.
 
 ---
+## Forks
+### Which Fork Should I Use?
+> [!WARNING]
+> If you have to ask, you're not ready for forks.
+
+Begin your research in Discord #custom-forks
+
+Please do not ask about forks outside of that channel.
+
+Please start with `commaai/master-ci`. This is the latest upstream and supports lateral only. 
+
+> [!WARNING]
+> Some forks will brick your C3X.
+> Most forks are not made for C3X because C3X is less than a year old.
+> 
+> Some forks contain banned code.
+> Using it will get you banned by the comma.ai company.
+> 
+> Some forks have nudgeless-lane-change.
+> Simply clicking the turn signal will move your car to the next lane.
+> Without any checks.
+> Yes, it will drive into the car next to you.
```

Comment:
```suggestion
> Some forks have nudgeless-lane-change. \
> Simply clicking the turn signal will move your car to the next lane. \
> Without any checks. \
> Yes, it will drive into the car next to you.
```